starfish/core/segmentation_mask.py

```diff
@@ -154,6 +170,38 @@ def from_label_image(
 
         return cls(masks)
 
+    def to_label_image(
+            self,
+            shape: Optional[Tuple[int]] = None,
```

Comment:
```suggestion
            shape: Optional[Tuple[int, ...]] = None,
```

Reply:
Would it be accurate to describe `shape` as kind of a crop operation?

Reply:
Ah, that's not what I was thinking of it as, and it would probably error if so. I was thinking more of when someone wanted to superimpose it on the original image so the shapes would match. Is this useful functionality?

Reply:
ohhh, i get it.  this _is_ useful functionality.